src/view/misc/article_licensing.jsx

```diff
@@ -82,16 +82,21 @@ class ArticleLicensing extends Component {
                 <div>
                   <h6>{licensedTitle}</h6>
                   <p>
-                    {Object.keys(licenses).map((name) => (
-                      <a
-                        rel="noopener"
-                        target="_blank"
-                        title={name}
-                        class={licenses[name].icon ? 'icon' : ''}
-                        href={licenses[name].url}>
-                        {licenses[name].icon ? <i class={licenses[name].icon}></i> : name}
-                      </a>
-                    ))}
+                    {Object.keys(licenses).map((name) =>
+                      (Array.isArray(licenses[name].icon)
+                        ? licenses[name].icon
+                        : [licenses[name].icon]
+                      ).map((icon) => (
```

Comment:
It seems that you are creating separate links for each icon. Can you include all icons of a `license` in a single link while keeping the visuals unchanged?

Reply:
I've tried, but it need more CSS-tunning of `hexo-theme-icarus`. I'll cehck it out later.